src/runtime/server.ts

```diff
@@ -44,15 +47,18 @@ global["browser"] = false;
 
 class HTTPRuntimeClient extends RuntimeClient {
   server: ServerDatabase;
-  constructor() {
+  constructor(databases) {
     let server = new ServerDatabase();
     const dbs = {
       "http": new HttpDatabase(),
       "server": server,
       "shared": shared,
       "browser": new Database(),
     }
-    super(dbs);
+    var uniondb = {}
```

Comment:
had no idea, i'll scrub my vars...why did B.E. make such a bad scoping...i guess it was supposed to be for lua-like use cases